specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2021-02-10/service.json

```diff
@@ -2997,28 +3318,101 @@
         }
       }
     },
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{resourceName}/replicationFabrics/{fabricName}/replicationProtectionContainers/{protectionContainerName}/replicationProtectedItems/{replicatedProtectedItemName}/failoverCancel": {
+      "post": {
+        "tags": [
+          "ReplicationProtectedItems"
+        ],
+        "summary": "Execute cancel failover.",
+        "description": "Operation to cancel the failover of the replication protected item.",
+        "operationId": "ReplicationProtectedItems_FailoverCancel",
```

Comment:
Do we support cancel failover?

Reply:
Yes cancel failover is supported in V2A RCM. I will add examples for it, once this PR is merged.

Reply:
@imshashwataggarwal: Can you please add examples. Currently some validations are failing because of this. This will help in surfacing new validation errors if any.
check PR: https://github.com/Azure/azure-rest-api-specs/pull/13287

Reply:
added

Reply:
yes it is added by @imshashwataggarwal .
Resolving comment

---
In reply to: [587894998](https://github.com/AsrOneSdk/azure-rest-api-specs/pull/16#discussion_r587894998) [](ancestors = 587894998)

Reply:
added examples. resolving comment

---
In reply to: [588004248](https://github.com/AsrOneSdk/azure-rest-api-specs/pull/16#discussion_r588004248) [](ancestors = 588004248)

Reply:
done

---
In reply to: [590333397](https://github.com/AsrOneSdk/azure-rest-api-specs/pull/16#discussion_r590333397) [](ancestors = 590333397)